integration_tests/elastic_search_test.go

```diff
@@ -0,0 +1,55 @@
+package integration_test
+
+import (
+	. "github.com/onsi/ginkgo"
+
+	"github.com/alphagov/paas-compose-broker/integration_tests/helper"
+	"os"
+)
+
+var _ = Describe("Broker Compose Integration: ElasticSearch", func() {
```

Comment:
Can we use `Describe` and [`Context`](https://onsi.github.io/ginkgo/#structuring-your-specs) here? It feels more idiomatic.